src/queries.js

```diff
@@ -127,13 +130,23 @@ export const articleInfoFragment = gql`
 
 export const subjectQuery = gql`
   ${topicInfoFragment}
-  query subjectQuery($subjectId: String!) {
+  query subjectQuery($subjectId: String!, $filterIds: String) {
     subject(id: $subjectId) {
       id
       name
       path
-      topics(all: true) {
+      topics(all: true, filterIds: $filterIds) {
```

Comment:
mm, we should not use both all and subtopics here. Should probably only use all and change make function similar to getSubjectMenu selector.